src/main/java/pl/javastart/bootcamp/domain/user/role/UserRoleRepository.java

```diff
@@ -0,0 +1,13 @@
+package pl.javastart.bootcamp.domain.user.role;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.transaction.annotation.Transactional;
+
+public interface UserRoleRepository extends JpaRepository<UserRole, Long> {
+    @Transactional
+    @Modifying
+    @Query(value = "delete from UserRole u where u.user.id = :id and u.role = :role")
+    void dismissRole(Long id, Role role);
```

Comment:
Możesz zostawić tak jak jest, ale w ogólności starałbym się nie delegować logiki aplikacji jako zapytań bazodanowych, tylko lepiej jest ją zachować jako zwykły kod Javy w klasach XyzService. Realizacja logiki przez specyficzne zapytania SQL/JPQL ma sens np. w sytuacji, gdy jesteśmy w ten sposób coś wyraźnie zoptymalizować względem tego, jak skomplikowane byłoby to w kodzie Javy, ale w tym przypadku to akurat nie jest problem.